tests/wpt/mozilla/tests/mozilla/mozbrowser/mozbrowsersecuritychange_event.html

```diff
@@ -0,0 +1,49 @@
+<head>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+</head>
+<body>
+<script>
+
+async_test(function(t) {
+
+  const HTTP_URL = "http://web-platform.test:8000";
+  const HTTPS_URL = "https://web-platform.test:8443";
+
+  var urls = [ HTTP_URL, HTTPS_URL, HTTP_URL ];
+  var url_index = 0;
+
+  var expectedEvents = [
+    {state: "insecure"},
+    {state: "secure"},
+    {state: "insecure"},
+  ];
+
+  var receivedEvents = [];
+
+  var iframe = document.createElement("iframe");
+  iframe.mozbrowser = "true";
+  iframe.src = urls[url_index++];
+
+  iframe.addEventListener("mozbrowsersecuritychange", t.step_func(e => {
```

Comment:
Let's make a separate `async_test` for each iframe so we get better granularity for test failures.